src/pybind/mgr/dashboard/frontend/src/styles.scss

```diff
@@ -1,9 +1,9 @@
 /* You can add global styles to this file, and also import other style files */
 @use './src/styles/defaults' as *;
+@use '@carbon/colors';
```

Comment:
This is not needed here, right?
```suggestion
```

Reply:
I will check and remove if it's not using

Reply:
I will check and remove if it's not using